brainscore/submission/evaluation.py

```diff
@@ -55,8 +55,10 @@ def run_evaluation(config_file, work_dir, db_connection_config, jenkins_id, mode
         for model in test_models:
             function = lambda: module.get_model(model)
             base_model_pool[model] = LazyLoad(function)
-            if module.get_layers(model) is not None:
+            try:
                 layers[model] = module.get_layers(model)
+            except Exception:
+                pass
```

Comment:
Maybe add a `logging.warning(f'Could not retrieve layer for model {model} -- skipping model')` to make things more transparent